tests/test_condaforge_config_schema.py

```diff
@@ -1,8 +1,7 @@
 import pytest
 from pydantic import ValidationError
-import yaml
-from conda_smithy.schema import ConfigModel
 
+from conda_smithy.schema import ConfigModel
```

Comment:
Interesting here it groups some `from...import...`s with `import`s and splits some out. Or is it just trying to group external `import`s separately from internal ones?

Reply:
This is exactly what happens.